client/src/modules/stock/lots/registry.service.js

```diff
@@ -187,6 +187,37 @@ function LotsRegistryService(uiGridConstants, Session) {
       headerCellFilter : 'translate',
       cellFilter : 'translate',
       visible : false,
+    }, {
+      field : 'is_asset',
+      displayName : 'FORM.LABELS.ASSET',
+      headerCellFilter : 'translate',
+      visible : false,
+      cellTemplate : '/modules/inventory/list/templates/asset.cell.tmpl.html',
+    }, {
+      field : 'reference_number',
+      displayName : 'FORM.LABELS.REFERENCE_NUMBER',
+      headerCellFilter : 'translate',
+      visible : false,
+      width : 150,
+      // ??? cellTemplate : '/modules/inventory/list/templates/reference_number.cell.html',
```

Comment:
You can remove this line I think the current way is okay without this template.

Reply:
Done